src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -98,6 +98,11 @@ public CommandResult execute(Model model) throws CommandException {
         personToEdit.setTags(updatedTags);
         model.setPerson(personToEdit, personToEdit);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        if (editPersonDescriptor.getName().isPresent()) {
```

Comment:
Nit: Could you help make it clearer what this condition is checking for?

```suggestion
        boolean isNameEdited = editPersonDescriptor.getName().isPresent();
        if (isNameEdited) {
```

Reply:
❓To clarify: I've checked the file history for these lines of code on IntelliJ IDEA (after running `gh pr checkout` for both this PR and #89) and I can't seem to find where this was fixed?

I note this PR has already been merged. I'll add a link to this comment thread on open PR #89 (where these changes have been merged to the PR as part of commit [5097e64](https://github.com/AY2526S1-CS2103T-F10-1/tp/pull/89/commits/5097e64be3b909b9dbcdaafec9d0946b577dc3fe). Let's continue discussion there if needed.